### PR TITLE
PMM-6212 Build dbaas-tools as well

### PR DESCRIPTION
**Pull request by Sudokamikaze**

### CODE REVIEW

File: pmm/pmm2-server-autobuild.groovy [hunk 1]
```diff
@@ -118,6 +118,7 @@ pipeline {
                             build-server-rpm pmm-server
                             build-server-rpm pmm-update
                             build-server-rpm dbaas-controller
```
Comment (AlekSi) [inline pmm/pmm2-server-autobuild.groovy:120]: Please check that we have that line in other places too. I think we miss it somewhere

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
> build-server-rpm pmm-manage

How that could work?!

**Comment by Sudokamikaze** [comment]
> > build-server-rpm pmm-manage
> 
> How that could work?!

You tell me :))

**Comment by AlekSi** [comment]
Ah, that's #540

**Comment by AlekSi** [comment]
And #525

**Review by AlekSi — APPROVED** [review]

**Pull request merged**